mycli/packages/special/iocommands.py

```diff
@@ -331,3 +332,45 @@ def unset_once_if_written():
     global once_file
     if written_to_once_file:
         once_file = None
+
+
+@special_command(
+    'watch',
+    'watch [seconds] query',
+    'Executes the query every [seconds] seconds (by default 5).'
+)
+def watch_query(arg, **kwargs):
+    usage = "Syntax: watch [seconds] query.\n"
+    if not arg:
+        yield (None, None, None, usage)
+        raise StopIteration
+    try:
+        args = arg.split(' ')
+        seconds = float(args[0])
+        statement = " ".join(filter(None, args[1:])).strip()
+    except ValueError:
+        seconds = 5
+        statement = arg
+    if not statement:
+        yield (None, None, None, usage)
+        raise StopIteration
+    cur = kwargs['cur']
+    sql_list = [
+        (sql.rstrip(';'), '> %s' % (sql))
```

Comment:
Can we use the new-style string formatting (i.e. `str.format()`) instead of the old-style (i.e. `%`-formatting) here?

Reply:
Oops, that slipped in from a copy&paste 😅 of course, I'll fix it in the next commit.